CONTRIBUTING.md

```diff
@@ -2,20 +2,20 @@
 
 ## Code of Conduct
 
-We are committed to providing a friendly, safe and welcoming environment for all. Make sure to take a look at the [Code of Conduct](CODE_OF_CONDUCT.md)!
+We are committed to providing a friendly, safe and welcoming environment for all. See our [Code of Conduct](CODE_OF_CONDUCT.md) for details.
```

Comment:
I know what we want to achieve with "Make sure to take a look" but I think very few people look at it and very few break the rules anyway. The type of person that breaks the rules is definitely not the type of person that reads the code of conduct before engaging with us, so we should not encourage a burden on nice people.

Reply:
I think it's just a different cultural way of saying the same thing. Your phrasing is friendlier.